src/main/java/uk/gov/ida/matchingserviceadapter/configuration/EuropeanIdentityConfiguration.java

```diff
@@ -30,10 +37,11 @@ public String getHubConnectorEntityId() {
         return hubConnectorEntityId;
     }
 
-    public String[] getAcceptableHubConnectorEntityIds() {
-        return acceptableHubConnectorEntityIds != null && acceptableHubConnectorEntityIds.length > 0
-            ? acceptableHubConnectorEntityIds
-            : new String[] { hubConnectorEntityId };
+    public List<String> getAcceptableHubConnectorEntityIds(MatchingServiceAdapterEnvironment environment) {
+        Set<String> entityIds = new HashSet<>(DEFAULT_ACCEPTABLE_HUB_CONNECTOR_IDS.getOrDefault(environment, new ArrayList<>()));
```

Comment:
Can `environment` be retrieved within this method instead of passed in?

Reply:
Maybe with injection? I'll look into it...

Reply:
This really does not seem trivial.